src/SDKs/Monitor/Monitor.Tests/Monitor.Tests.csproj

```diff
@@ -5,13 +5,31 @@
     <Description>Monitor.Tests Class library</Description>
     <PackageId>Monitor.Tests</PackageId>
     <VersionPrefix>1.0.0-preview</VersionPrefix>
-    <TargetFrameworks>netcoreapp1.1</TargetFrameworks>
   </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452</TargetFrameworks>
+    <!-- TargetFrameworks>net452;netcoreapp1.1</TargetFrameworks -->
+  </PropertyGroup>
+
+  <ItemGroup>
+	  <!-- PackageReference Update="Microsoft.NET.Test.Sdk" Version="15.0.0" / >
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.8" / >
+	  <PackageReference Update="Microsoft.Rest.ClientRuntime.Azure.TestFramework" Version="1.7.2" / -->
+    <PackageReference Include="Newtonsoft.Json" Version="6.0.8" />
```

Comment:
That downgraded version comes from AzSdk.reference.props, the item group for net452. It uses 9.0.1 for netstandard1.4. My tests are targeting net452.
Without the explicit reference in the test project all my tests fail in VS with an error that says: Could not load file or assembly 'Newtonsoft.Json, Version=6.0.0.0.... I think this is so since the clientruntime depends on 6.0.8 for net452 (and on 9.0.1 for netstandard1.4.)